### DISCUSSION TITLE
Fitbit watch would not update states when phone's bluetooth is on

### DISCUSSION THREAD

**Discussion by alexbell62** [mKeRix/room-assistant#364] [Q&A]
I noticed that this fitness watch `Fitbit Versa Health` stops reporting rooms when the phone it is linked to turns on bluetooth (even if no device connected to that bluetooth) leading to getting status 'not home'. 

Another Fitbit watch in the house `Fitbit Sense` works perfectly with the same set up with bluetooth on or off. 

 I wonder if anyone had the same issue?? I saw in discussions about somewhat similar issues with the apple watch here and wonder if anything can be done with fitbit as well?

**Comment by Swanky-Bubbles** [comment]
Same issue here with Fitbit Ionics

I read somewhere that Fitbit disables BLE when paired with a phone, not sure how to get around that yet

**Comment by alexbell62** [comment]
Hi @dkramer74 

Please let me know if you find out workarounds. 
What i noticed last few days it is not specific to model of the watch as i wrote earlier. Now my sense doesnt work while versa is doing more or less alright.

I think the troubles start when payment card added to the watch. Even if removed afterwards.. 
I also noticed that after restart th ewatch sends ble for some time but then stops dioing that.

**Reply by Swanky-Bubbles** [reply]
Hey @alexbell62 

I found a workaround that seems to work on my Ionic, not sure about other models.

I went to the bluetooth settings on the watch, all the way at the bottom I have "Restore Music Controls" Selected "Pair" there then paired it on my phone. Then set it up on Room Assistant using Bluetooth Classic integration and the separate classic MAC that was displayed on my phone pairing screen.

Hope that works out for you too

**Reply by alexbell62** [reply]
Cool thanks so much for this! 
I wonder if it is going to work with raspberry pi zero w? I also use it for reading ble signals from xiomy sensors so i wonder if rpi zero w can read both ble and bluetooth classic simultaneously?

**Reply by alexbell62** [reply]
nah didt work for me. Looks like fitbit sense stops sending ble and bluetooth signals some time after reboot. I wonder if it is a bug in my watches?
Otherwise all works good for versa watch...

**Reply by Swanky-Bubbles** [reply]
Well it looks like it only worked for me for about half a day...

**Comment by JesusRo** [comment]
my 2 cents here,
I had same behaviour with a xiaomi band and a Gear S3. If they connect to the phones, they just stop sending BLE
My workaround was to macro my phone to disable bluethoot at home (and re-enable it outside), forcing my watch to send the BLE signals again and connect to the wifi for the notifications.
For xiaomi band, same but as it has no wifi, no notifications at home

**Reply by Swanky-Bubbles** [reply]
I just set this up via tasker, seems to be working out ok so far

**Reply by alexbell62** [reply]
Thanks for the idea @JesusRo!
@dkramer74 just wonder how did you configure tasker? Did you set it up to switch off bluetooth when you are home?
And do you test it with ble or bluetooth classic trackers?

Would love to know how it goes!

**Reply by Swanky-Bubbles** [reply]
I set tasker up to just disable BT when I connect to my home WiFi SSID

**Comment by phate1** [comment]
Just installed room assistant and trying to use a first version versa and just found exactly the same thing as soon as phone Bluetooth is on stops sending out ble beacons. It doesn't use wi-fi for notifications so Bluetooth off stops watch working :( assume nobody found any other work arounds since ? Cheers
